Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Components/Pages/BulkPagesController.cs

```diff
@@ -78,9 +78,13 @@ public BulkPageResponse AddBulkPages(BulkPage page, bool validateOnly)
 
                 try
                 {
-                    string errorMessage = null;
-
-                    if (DuplicateExists(currentIndex, pages))
+                    string errorMessage = null;
+
+                    if (currentIndex == 0 && oTab.Level > 0)

```

Comment:
For validation part, 
else if (validateOnly)
will be executed and it will not hit else part.
However, the else part will be executed and show error on "Add Page(s)" flow.

Secondly, if we do not allow child to be the first line (in which case it will always be a parent) the above mentioned hierarchy will become valid.

Reply:
@bdukes 

> Secondly, if we do not allow child to be the first line (in which case it will always be a parent) the above mentioned hierarchy will become valid.

This makes sense to me, are you ok with this as is or I misunderstood something ?

Reply:
Yeah, now that I review it again it seems like it should work fine for what it's doing.  Thanks for reviewing @valadas